src/templates/base/2019/base.html

```diff
@@ -203,6 +203,17 @@
             </li>
           </ul>
         </nav>
+        <nav id="mobile-footer-nav-items" aria-label="{{ self.footer_title() }}" class="nav-items">
+          <ul>
+            {{ self.non_chapter_nav_links() }}
+            <li>
+              {{ year_switcher('mobile') }}
+            </li>
+            <li>
+              {{ language_switcher('mobile') }}
```

Comment:
Getting duplicate ids in the generate HTML so can we change this:

```suggestion
              {{ year_switcher('mobile-footer') }}
            </li>
            <li>
              {{ language_switcher('mobile-footer') }}
```